src/nn/ops/clamp.ts

```diff
@@ -1,34 +1,20 @@
 import * as tf from '@tensorflow/tfjs-core';
 
 import {MLClampOptions} from '../graph_builder';
-import {ConstantOperand, MLOperand, OutputOperand} from '../operand';
+import {MLOperand, OutputOperand} from '../operand';
 import {SingleOutputOperation} from '../operation';
 import * as utils from '../utils';
 
 export class Clamp extends SingleOutputOperation {
```

Comment:
As `clamp` only takes one input know, I would suggest to inherit from `UnaryMLOperator`.

Reply:
OK, I will update it, thanks @huningxin